cmd/deploy.go

```diff
@@ -152,11 +161,12 @@ func InitRunCommand() (*cobra.Command, error) {
 			concurrencyDefault = 0
 		}
 	}
-	var concurrency int
-	run.Flags().IntVarP(&concurrency, "concurrency", "c", concurrencyDefault, "concurrency")
+	run.Flags().IntP("concurrency", "c", concurrencyDefault, "concurrency")
 
-	var clusterUrl string
-	run.Flags().StringVar(&clusterUrl, "url", os.Getenv("KUBERNETES_CLUSTER_URL"),
+	run.Flags().String("url", os.Getenv("KUBERNETES_CLUSTER_URL"),
 		"URL of the Kubernetes cluster. Overrides KUBERNETES_CLUSTER_URL env variable in AppController pod.")
+
+	run.Flags().StringP("replicas", "n", "1", "number of flow replicas: [+]number")
```

Comment:
No, the whole point is that it's string so that you. See the difference:
kubeac run -n2 // create 2 replica of default flow
kubeac run -n+2 // increase replica count of default flow by two
kubeac run -n0 // delete all replicas (i.e. delete all resources)
kubeac run -n+0 // refresh resources (no additional replicas will be created)
if it would be int, +2 and 2 would be the same